<?xml version="1.0" ?>
<launch>

  <arg name="urdf_dir"
       default="$(find o2ac_parts_description)/urdf/generated"/>

  <node name="model_spawner" pkg="aist_model_spawner" type="model_spawner.py"
	output="screen">
    <param name="urdf_dir"	value="$(arg urdf_dir)"/>
  </node>

</launch>
